src/core/core.tooltip.js

```diff
@@ -562,10 +604,9 @@ var exports = Element.extend({
 		// that does _view = _model if ease === 1. This causes the 2nd tooltip update to set properties in both the view and model at the same time
 		// which breaks any animations.
 		var existingModel = me._model;
-		var model = me._model = getBaseModel(opts);
 		var active = me._active;
-
 		var data = me._data;
+		var i, len;
```

Comment:
we usually put unitialized variables after all initialized variables, so I might move this back to where it was originally